src/utils/validatePassword.js

```diff
@@ -0,0 +1,11 @@
+const validatePassword = (password) => {
+  if (!password) {
+    return 'password is required';
+  }
+
+  if (password.length < 6) {
+    return 'at least 6 characters';
+  }
+};
```

Comment:
Consider returning a default value (such as null or undefined) when the password is valid. This will make it clear that the function always returns a value, which can be useful for consistency in error handling.